button-card.js

```diff
@@ -242,34 +443,48 @@
       return 3;
     }
 
-    _toggle(state, config) {
-      switch (config.action) {
-        case 'toggle':
-          this.hass.callService('homeassistant', 'toggle', {
-            entity_id: state.entity_id,
-          });
-          break;
-        case 'more_info': {
-          const node = this.shadowRoot;
-          const options = {};
-          const detail = { entityId: state.entity_id };
-          const event = new Event('hass-more-info', {
-            bubbles: options.bubbles === undefined ? true : options.bubbles,
-            cancelable: Boolean(options.cancelable),
-            composed: options.composed === undefined ? true : options.composed,
-          });
-          event.detail = detail;
-          node.dispatchEvent(event);
-          return event;
+    _handleTap(state, config) {
+      if (config.tap_action) {
+        let event;
+        switch (config.tap_action.action) {
+          case 'none':
+            break;
+          case 'more-info':
+          case 'more_info':
```

Comment:
I think we should just deprecate `more_info` now. I'm going to merge this request and then do that real quick in a follow-up